unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/BodyShapeController.cs

```diff
@@ -66,6 +72,12 @@ public void SetupEyes(Material material, Texture texture, Texture mask, Color co
 
     public void SetupEyebrows(Material material, Texture texture, Color color)
     {
+        if (assetContainer?.transform == null)
+        {
+            Debug.LogWarning("Tried to setup eyes when the asset not ready");
+            return;
+        }
```

Comment:
remember not to overwrite the changes in PR #1461 with these :P

Reply:
This shouldn't be here. We had a bit of a fiesta with the branches while doing tests hahaha. I force pushed putting the branch up to master, really nice catch!